index.html

```diff
@@ -2,90 +2,129 @@
-          of Product & Engineering.
+          As a budding fullstack web developer, I bring great enthusiasm, charisma, potential and an insatiable Interest
+          in emerging technologies.
         </p>
       </div>
     </div>
-    <div class="divider">Projects</div>
+
+    <div class="divider">General Projects Delivered</div>
     <div class="section projects">
       <div class="project">
         <div class="explanation">
-          <p>Animated floating Hal Jordan, the Green Lantern.</p>
-          <p>Image is an SVG created in Sketch.</p>
-          <p>Animation is achieved with CSS.</p>
-          <a href="https://github.com/jlcarmic/green-lantern" target="_blank">
-            <button class="button">Green Lantern</button>
+          <p>Below is an overview of the various nature of projects I have had the pleasure of working on.</p>
+          <p>Please click the respective button below,</p>
+          <p>to visit some of my most recent works.</p>
+
+          <a href="https://github.com/Pahtreaq" target="_blank">
+            <button class="button">Web development</button>
+          </a>
+          <inline></inline>
+          <a href="https://www.facebook.com/pg/mewedge.inc/videos/" target="_blank">
+            <button class="button">Motion Graphics</button>
+          </a>
+          <a href="https://www.facebook.com/pg/mewedge.inc/posts/" target="_blank">
+            <button class="button">Desktop Graphics</button>
           </a>
         </div>
-        <div class="screenshot">
-          <img src="images/greenlantern.png" />
-        </div>
+        <div class="photo4"></div>
+        <img src="/images/h.png" />
       </div>
+    </div>
+
 
+    <div class="divider">MySql Project Delivered</div>
+    <div class="section projects">
       <div class="project">
-        <div class="screenshot">
-          <img src="./images/producthuntalexa.png" />
-        </div>
+        <div class="photo3"></div>
         <div class="explanation">
-          <p>An Alexa Skill for hearing Today's Featured products.</p>
-          <p>Built using NodeJS and AWS Lambda.</p>
-          <p>Utilizes ProductHunt's API.</p>
-          </p>
-          <a href="https://github.com/jlcarmic/producthunt_alexa_skill" target="_blank">
-            <button class="button">Today's Featured</button>
+          <p>Checkout my baseball database with the capability of pulling all the team stats and data.</p>
+          <p>This database enables various processes with the capability to retrieve teams.</p>
+          <p>Through this database, data can be extrapolated in relation to data.</p>
+          <a href="https://github.com/Pahtreaq/baseball-database/branches" target="_blank">
+            <button class="button">Check it Out</button>
           </a>
+          <div class="screenshot"></div>
+          <img class="Interests" src="./images/5.svg" />
```

Comment:
This image is showing up as broken when I clone locally

![image](https://user-images.githubusercontent.com/5775716/70354144-ea865d00-183c-11ea-8b6e-8a9b16371860.png)